examples/run/alpaka/seeding_example_alpaka.cpp

```diff
@@ -43,6 +43,10 @@
 #include "detray/propagator/propagator.hpp"
 #include "detray/propagator/rk_stepper.hpp"
 #include "traccc/alpaka/utils/vecmem_types.hpp"
+#ifdef ALPAKA_ACC_SYCL_ENABLED
+#include <CL/sycl.hpp>
```

Comment:
Please use

```suggestion
#include <sycl/sycl.hpp>
```

instead. I'll soon update the project to use this (SYCL2020) convention. So that we could use oneAPI 2025.0.0.